mountpoint-s3/tests/fstab/common.sh

```diff
@@ -0,0 +1,53 @@
+function build_mountpoint() {
+  build_out=$(cargo build --bin mount-s3 --release --features=fstab --message-format=json-render-diagnostics)
+
+  MOUNTPOINT_ROOT=$(dirname "$(which "$0")")/../../..
+  MOUNTPOINT_PATH=$(printf "%s" "$build_out" | jq -js '[.[] | select(.reason == "compiler-artifact") | select(.executable != null)] | last | .executable')
+  echo "Mountpoint path: $MOUNTPOINT_PATH"
+}
+
+function spawn_mounts() {
```

Comment:
So this means we're just starting Mountpoint on already running system, I also remember that fstab also allows you to mount filesystems during startup. Are there big differences between mounting at startup vs. mounting at runtime? Should we also have tests for startup as well?

Reply:
Another thing I remember is systemd/fstab is not behaving well if filesystems starts to fail during startup, and I think there was an option to not fail mount to allow system to start, should we also add test-case for that?

Reply:
Github's CI doesn't allow you to reboot the host during CI, so this is the best we can do without significant effort. From my testing on Fedora, you don't actually need `nofail` to make sure the system boots even with bad Mountpoint configuration, but I don't think it's worth tempting fate with the actual docs we're writing.